src/components/layout/Layout.jsx

```diff
@@ -36,39 +32,28 @@ const Layout = ({
           <html lang="en" />
         </Helmet>
         <SiteHeader
-          logo={headerLogo || EdXLogo}
-          logoDestination={`${process.env.BASE_URL}/${pageContext.enterpriseSlug}`}
-          logoAltText={pageContext.enterpriseName}
+          logo={enterpriseConfig.branding.logo || EdXLogo}
+          logoDestination={`${process.env.BASE_URL}/${enterpriseConfig.slug}`}
+          logoAltText={`${enterpriseConfig.name} logo`}
           loggedIn={!!username}
           username={username}
           avatar={profileImage.imageUrlMedium}
           mainMenu={getHeaderMenuItems('mainMenu')}
           userMenu={getHeaderMenuItems('userMenu')}
-          loggedOutItems={[
```

Comment:
Are these intentionally gone?

Reply:
Yeah, since we require the user to be authenticated when using any part of the application, the header should never be rendered as a logged out state.

In the root index.jsx file:

```
initialize({
  requireAuthenticatedUser: true,
  // ...
});
```

Reply:
right right, sounds good